specification/common-types/resource-management/v3/types.json

```diff
@@ -619,6 +619,15 @@
     }
   },
   "parameters": {
+    "Password": {
+      "type": "string",
+      "x-ms-secret": true,
+      "x-ms-external": true,
+      "x-ms-mutability": [
+        "create",
+        "update"
+      ]
+    },
```

Comment:
this file is not in our repo. I suggest we keep password within our file common-types.json